.vscode/extentions.json

```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"quarto.quarto",
+		"streetsidesoftware.code-spell-checker",
```

Comment:
Do you think this one is necessary here? We won't be writing too much code.

Reply:
It's also just a general spell checker. For example, if I write "hrello" in an `.md` file, CSpell will show that under problems as an "Unknown word"

![image](https://github.com/user-attachments/assets/1006b6ad-cc9f-4189-be14-e0bd835dbef7)